packages/reporters/cli/src/bundleReport.js

```diff
@@ -16,9 +17,14 @@ const COLUMNS = [
   {align: 'right'}, // time
 ];
 
-export default function bundleReport(bundleGraph: BundleGraph) {
+export default async function bundleReport(
+  bundleGraph: BundleGraph,
+  fs: FileSystem,
+  projectRoot: FilePath,
+) {
   // Get a list of bundles sorted by size
-  let {bundles} = generateBundleReport(bundleGraph);
+  // $FlowFixMe not entirely sure why this fails?
```

Comment:
You access `bundle.totalAssets`  but `totalAssets` does't exist in `BundleStats`.